### DISCUSSION TITLE
FutureWarning Jupyter Notebook

### DISCUSSION THREAD

**Discussion by elvikha** [teamalgoritma/community#11] [Q&A] [answered]
Ini maksudnya apa ya?
![image](https://user-images.githubusercontent.com/101514469/158933744-d797752b-4b64-4040-94be-d58306d059b7.png)

**Comment by onlyphantom** [comment, marked as answer]
Warning itu tipe FutureWarning, warning yang bermaksud “In the future, this code should be replaced by something else (deprecated)” 

In simple words, “Jupyter uses ipykernel, and in the current version of ipykernel it uses `numeric_only=None`. This is OK now but in a future version it’s not OK.”

There’s nothing to worry about as a user, but it’s probably something the team behind ipykernel needs to be aware of 😀
